cves/CVE-2016-8740.yml

```diff
@@ -5,6 +5,8 @@ CWE_instructions: |
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
 CWE:
+  The mod_http2 module in the Apache HTTP Server 2.4.17 through 2.4.23, when the Protocols configuration includes h2 or h2c, does not restrict request-header length, which allows remote attackers to cause a denial of service
```

Comment:
I'm a little confused with what this is. It should be the CWE of the CVE. Commonly in the format CWE-400 or just simply 400